.drone.yml

```diff
@@ -234,6 +234,7 @@ steps:
         from_secret: SAUCE_USERNAME
       SAUCE_ACCESS_KEY:
         from_secret: SAUCE_ACCESS_KEY
+    shm_size: '2gb' # necessary to avoid page crashes in Selenium
```

Comment:
I wouldn't mind a comment (or a link back to a GH comment on this PR!) showing what these look like, in case somebody in the future needs to re-evaluate and say bump it up to 3gb or whatever.